splinterd/api/static/openapi.yaml

```diff
@@ -342,6 +342,61 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
 
+  /authorization/roles:
+    get:
+      summary: Fetches a list of roles
```

Comment:
We should add a tag or two to group the RBAC and possibly all authorization endpoints together.